src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionFixAllTests.cs

```diff
@@ -160,54 +164,59 @@ int M(int i, int j)
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
         public async Task TestNested_02()
         {
-            await TestInCSharp8(
-@"class Program
+            var input = @"class Program
 {
-    System.Action<int> M(int i, int j)
+    System.Func<int> M(int i, int j)
     {
-        {|FixAllInDocument:switch|} (i)
+        [|switch|] (i)
         {
             // 1
             default: // 2
                 return () =>
                 {
-                    switch (j)
+                    [|switch|] (j)
                     {
                         default:
                             return 3;
                     }
                 };
         }
     }
-}",
-@"class Program
+}";
+            var expected = @"class Program
 {
-    System.Action<int> M(int i, int j)
+    System.Func<int> M(int i, int j)
     {
         return i switch
         {
             // 1
             // 2
             _ => () =>
                               {
-                                  switch (j)
+                                  return j switch
                                   {
-                                      default:
-                                          return 3;
-                                  }
+                                      _ => 3,
+                                  };
                               }
 
             ,
         };
     }
-}");
+}";
+
+            await new VerifyCS.Test
+            {
+                TestCode = input,
+                FixedCode = expected,
+                NumberOfFixAllIterations = 2,
```

Comment:
📝 This always means the Fix All fails to fix _all_ (in one invocation). The `expected` output was achieved via execution of the Fix All operation twice in sequence.